ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_edit.robot

```diff
@@ -0,0 +1,65 @@
+*** Settings ***
+Documentation      Suite to test Data Science Projects (a.k.a DSG) feature aimed on editing the existing instances
+Library            SeleniumLibrary
+Library            OpenShiftLibrary
+Resource           ../../../Resources/OCP.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource
+Suite Setup        Project Suite Setup
+Suite Teardown     Project Suite Teardown
+Test Setup         Launch Data Science Project Main Page
+Test Teardown      Close All Browsers
+
+
+*** Variables ***
+${PRJ_TITLE}=                  ODS-CI DS Project
```

Comment:
ah probably it would be a good thing to have a different name for the project, in order to avoid issues with other suites in case the teardown fails for some reasons.

Reply:
Thank you, Berto, updated :slightly_smiling_face: 

I also wanted to re-run the tests locally, just to be sure, but I'm having some issue with my cluster at the moment.